sentry-android-core/src/main/java/io/sentry/android/core/ManifestMetadataReader.java

```diff
@@ -84,8 +84,8 @@ static void applyMetadata(
         options.setEnableSessionTracking(sessionTrackingEnabled);
 
         if (options.getSampleRate() == null) {
-          // TODO: it needs to read a Float I guess
-          Double sampleRate = metadata.getDouble(SAMPLE_RATE, -1);
+          // manifest meta-data only reads floats
```

Comment:
`l`: Maybe it makes sense to add this link here: https://developer.android.com/guide/topics/manifest/meta-data-element

Reply:
I put it on the `Motivation and Context`